### ISSUE TITLE
Add null check for getSearchKeywords method on the AddressField

### ISSUE THREAD

**Issue by myleshyson** [doublesecretagency/craft-googlemaps#108]
Can you add a null check to `\doublesecretagency\googlemaps\fields\AddressField::getSearchKeywords`? If that field value is null, then when trying to re-save and update search indexes, that method will throw an exception.

I think you can just add something to the top like 

```php
 public function getSearchKeywords(mixed $value, ElementInterface $element): string
{
     if (!$value) {
         return '';
     }
    //...
}
```

**Comment by myleshyson** [comment]
And better can you add back the parent call so that people can hook into `EVENT_DEFINE_KEYWORDS` to customize how the address field keywords are indexed.

**Comment by lindseydiloreto** [comment]
Copy that, thanks for reporting. Out of curiosity, which version of Craft and/or Google Maps are you running?

**Comment by myleshyson** [comment]
No problem!

Craft 4.7.3
Google Maps 4.3.11

**Comment by lindseydiloreto** [comment]
Good news! Both of these changes are now available in [v4.5.1](https://github.com/doublesecretagency/craft-googlemaps/releases/tag/4.5.1). 👍 

See the docs regarding the [new event to define keywords](https://plugins.doublesecretagency.com/google-maps/address-field/event/).
